src/SpeedyWeather.jl

```diff
@@ -2,37 +2,37 @@ module SpeedyWeather
 
     # STRUCTURE
     import Parameters: @with_kw, @unpack
-    
+
     # NUMERICS
     import FastGaussQuadrature
     import AssociatedLegendrePolynomials
     import FFTW
     import Primes
     import LinearAlgebra
 
-    # INPUT OUTPUT 
+    # INPUT OUTPUT
     import Dates: Dates, DateTime
     import Printf: @sprintf
     import NetCDF: NetCDF, NcFile, NcDim, NcVar
     import BitInformation: round, round!
     import UnicodePlots
     import ProgressMeter
-    
+
     # EXPORT MAIN INTERFACE TO SPEEDY
     export run_speedy, initialize_speedy
 
     # EXPORT STRUCTS
-    export Parameters, GenLogisticCoefs,
+    export Parameters, GenLogisticCoefs, HumidityCoefs,
```

Comment:
I've exported the `GenLogisticCoefs` in case someone wants to change the vertical resolution. However, with many of the other structs, I don't think they need to be exported as I doubt people would use them frequently enough to justify the `GeneratorFunction` instead of `SpeedyWeather.GeneratorFunction` shortcut.